helper/resource.js

```diff
@@ -0,0 +1,140 @@
+const { ResourceType } = require("../common/enum.utils");
+const rp = require('request-promise');
+exports.getDistricts = async function(req,res){
+    return new Promise(async function (resolve, reject) {
+        try {
+            let bodyParam = gen.utils.getDruidQuery("distric_level_query");
+            if(req.query.resourceType == ResourceType.SOLUTION) {
```

Comment:
@ankitpws Please optimise this code, use ternary operator where possible and put comments.